src/Common/src/Interop/Windows/Interop.BOOLEAN.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal partial class Interop
+{
+    public struct BOOLEAN
```

Comment:
I realize this is nested within an internal type, so it will never actually be public, but I'm curious: should these nested types be `public` or `internal`? The visibility of `Interop` types in this PR are inconsistent. Is there an existing convention?

Reply:
Internal is better- I'll fix it